### PR TITLE
added handlebars

### PR DESCRIPTION
**Pull request by erdemkoral**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,30 +62,11 @@
       </select>
     </section>
     <section id="chickens" class="tab-content">
```
Comment (JPGothro) [inline index.html:64]: The arrangement of your filters in a separate section from the set of Chickens causes the filter drop-downs to always be displayed (incorrectly at the bottom of the About tab, and correctly at the top of the Coop tab). Take a look at the assigments we've worked on to see how to more properly arrange the filters and what the filters apply to.

File: app/app.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 var allChickens = [];
+var view ={};
```
Comment (JPGothro) [inline app/app.js:2]: While you obviously can put all this code in one file, it doesn't make it easier to read, manage, maintain. Haven't we reviewed MVC concepts? Better practice is to have separate files for the Model and View components.

File: app/app.js [hunk 3]
```diff
@@ -12,21 +13,10 @@ function Chicken (chix)  {
...
-  $newChicken.find('#4').text("Average Weight: " + this.weight);
-  $newChicken.find('.publishedOn').text("Published on " + this.publishedOn +" (" + daysAgo +" days ago)");
-  $newChicken.find('.description').html(this.description);
-  $newChicken.find('.head-shot img').attr('src',  this.image);
-
-  return $newChicken;
+  this.daysAgo = parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000);
+  var template = $('#chickenTemplate').html();
+  var compiledTemplate = Handlebars.compile(template);
+  $('#chickens').append(compiledTemplate(this));
```
Comment (JPGothro) [inline app/app.js:19]: This function is doing an append of the complied template data. There is also an append being done in the forEach() function on line 43. Ideally, you should only do it once (one is actually ignored), and the toHtml() function you should "return" the compiled Template data.

**Comment by erdemkoral** [comment]
I actually refactored that on my most recent portfolio assignment.
Thank you

**Comment by erdemkoral** [comment]
I also fixed this one, you can see in my most recent assignment.
Thanks

**Pull request merged**